R/00-Id.R

```diff
@@ -56,3 +57,36 @@ toString.Id <- function(x, ...) {
 dbQuoteIdentifier_DBIConnection_Id <- function(conn, x, ...) {
   SQL(paste0(dbQuoteIdentifier(conn, x@name), collapse = "."))
 }
+
+
+#' Order DBI::Id() user inputs
+#'
+#' Create the specific order:
+#' catalog > cluster > schema > table
+#'
+#' @param ... table param and unnamed `Id` components
+#' @param database optional database `Id` component
+#' @param catalog optional catalog `Id` component
+#' @param cluster optional cluster `Id` component
+#' @param schema optional schema `Id` component
+#' @param table optional table `Id` component
+#'
+#' @return vector of user's inputs, with names if specified, in correct SQL order
+#'
+#' @examples
+#' DBI:::orderIdParams(table = "flights", schema = "nycflights13", 'unnamed_id')
+#' DBI:::orderIdParams("nycflights13", "flights")
+#'
```

Comment:
I think you can drop these docs; the goal of this function is very clear to me and it's not exported.